GPU/GLES/GPU_GLES.cpp

```diff
@@ -610,14 +605,15 @@ void GPU_GLES::CheckGPUFeatures() {
 		features |= GPU_SUPPORTS_TEXTURE_FLOAT;
 
 	// If we already have a 16-bit depth buffer, we don't need to round.
-	if (fbo_preferred_z_bitdepth() > 16) {
+	bool prefer24 = draw_->GetDeviceCaps().preferredDepthBufferFormat == Draw::DataFormat::D24_S8;
```

Comment:
Well, not sure if there are any important devices using it, but the intention was to detect `D32F` as well.

-[Unknown]